test/theme_test.dart

```diff
@@ -1,24 +1,26 @@
+import 'package:flame_splash_screen/flame_splash_screen.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:flame_splash_screen/flame_splash_screen.dart';
-
 void main() {
   test('white', () {
     final theme = FlameSplashTheme.white;
-    expect(theme.constraints, BoxConstraints.expand());
+    expect(
+      theme.constraints,
+      const BoxConstraints.expand(),
+    );
     expect(
       theme.backgroundDecoration,
-      BoxDecoration(color: const Color(0xFFFFFFFF)),
+      const BoxDecoration(color: const Color(0xFFFFFFFF)),
     );
   });
 
   test('dark', () {
     final theme = FlameSplashTheme.dark;
-    expect(theme.constraints, BoxConstraints.expand());
+    expect(theme.constraints, const BoxConstraints.expand());
     expect(
       theme.backgroundDecoration,
-      BoxDecoration(color: const Color(0xFF000000)),
+      const BoxDecoration(color: const Color(0xFF000000)),
```

Comment:
```suggestion
      const BoxDecoration(color: Color(0xFF000000)),
```
do you have our linter on this? it would probably complain about the redundant internal const